src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/GeneralType.php

```diff
@@ -140,6 +140,23 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ),
                 'required' => false,
             ])
+            ->add('product_breadcrumb_category', ChoiceType::class, [
+                'label' => $this->trans(
+                    'Category used in breadcrumbs',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'choices' => [
+                    'Product default category' => 'default',
+                    'Category the product was accessed from' => 'current',
```

Comment:
```suggestion
                    'Product default category' => 'default',
                    'Category the product was accessed from' => 'current',
```

Those two keys should be translatable phrases

Reply:
ping @Dreimus :)

Reply:
@kpodemski, I just added the capability